confgenerator/agentmetrics.go

```diff
@@ -72,6 +72,7 @@ func (r MetricsReceiverAgent) Pipeline() otel.Pipeline {
 					// change data type from double -> int64
 					otel.ToggleScalarDataType,
 				),
+				otel.AddPrefix("agent.googleapis.com"),
```

Comment:
Add a TODO to fold this back into the `RenameMetric` calls? Also in other files…

Reply:
Don't we plan to use Rename for non-agent metrics?

Reply:
I'm not 100% convinced that's the right choice. It might be, but not sure.